app/src/test/java/org/oppia/app/testing/options/OptionsFragmentTest.kt

```diff
@@ -57,6 +59,28 @@ class OptionsFragmentTest {
     }
   }
 
+  @Test
+  fun clickAppLanguage_checkSendingTheCorrectIntent() {
```

Comment:
Ditto change for naming.

Also fix this for you StoryTextSize PR too.

Reply:
>Also fix this for you StoryTextSize PR too.

Done in #1579

Reply:
>Ditto change for naming.

Done